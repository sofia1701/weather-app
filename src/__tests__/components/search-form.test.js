import React from 'react';
import { render, cleanup } from "@testing-library/react";
import SearchForm from '../../components/search-form';

afterEach(cleanup);

describe("SeachForm", () => {
  it('renders correctly', () => {
    const { asFragment } = render(<SearchForm />);
  
    expect(asFragment).toMatchSnapshot();
  });
  it('renders text input', () => {
    const { getByTestId } = render(
      <SearchForm />
    )
    expect(getByTestId("input-id")).toHaveClass("input");
  })
  it('renders a button', () => {
    const { getByTestId } = render(
      <SearchForm />
    )
    expect(getByTestId("button-id")).toHaveClass("button");
  })
})