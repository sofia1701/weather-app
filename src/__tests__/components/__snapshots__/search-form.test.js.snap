// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SeachForm renders correctly 1`] = `[Function]`;
